packages/eas-cli/src/devices/__tests__/manager-test.ts

```diff
@@ -41,38 +42,38 @@ describe(AccountResolver, () => {
 
         const resolver = new AccountResolver(projectDir, user);
         const account = await resolver.resolveAccountAsync();
-        expect(account).toEqual(user.accounts[1]);
+        expect(account).toEqual(user.accounts![1]);
```

Comment:
All these `user.accounts!` notations are also a side effect of the `accounts: [Account]` GraphQL typing.